kong/db/strategies/cassandra/init.lua

```diff
@@ -1131,8 +1135,8 @@ do
           clear_tab(current_entity_ids)
           current_entity_count = 0
           for i, row in ipairs(rows) do
-            current_entity_ids[i] = row.entity_id
             current_entity_count = current_entity_count + 1
+            current_entity_ids[current_entity_count] = row.entity_id
```

Comment:
@rainest Just to make sure I'm reading this right: this change has no effect, right? `current_entity_count` will always equal `i` in this case. Was this made just so that the code looks like the one changed above (which does have an effect)?

Reply:
The Enterprise variant of this loop includes an additional condition for adding the entity, along the lines of:
```
for i, row in ipairs(rows) do
  if condition then
    current_entity_ids[i] = row.entity_id
    current_entity_count = current_entity_count + 1
  end
end
```
That condition allows for gaps in `current_entity_ids`, causing breakage further on. It does indeed have no effect without that additional condition.